### ISSUE TITLE
Deployment information

### ISSUE THREAD

**Issue by evan10s** [hackumass/hackerforce#13]
I'm trying to deploy this in production - do you have any information on what environment variables are required (for instance, for sending emails) and how/where those should be defined?  (When I try to view the wiki, I just get redirected back to the repo homepage)

**Comment by Tookmund** [comment]
Take a look at #11 I just filed it to try and document all the environment variables used

**Comment by evan10s** [comment]
That looks very helpful.  Thanks!

**Comment by ikhatri** [comment]
Hi folks, I'm going to re-open this issue because I want it to be a standing issue that we resolve (and that the community is welcome to contribute to). The documentation for hackerforce is currently not up to the standards I'd like it to be, mostly due to our focus on the [dashboard](https://github.com/fuseumass/dashboard) documentation & release for the few hackathons who had previously committed to using it.

That said I'd like to use the wiki for dashboard as a template for what we can do here with hackerforce. We should have deployment instructions that support a minimum configuration of Heroku + Amazon Simple Email Service (which is the configuration we currently use for HackUMass in production).

Basically to start with I think we should remove the [docs/](https://github.com/fuseumass/hackerforce/tree/development/docs) folder and switch to a git wiki for consistency. Unfortunately they don't support pull requesting, but we can still merge any changes from forks using [this](https://gist.github.com/larrybotha/10650410) method.

@Tookmund if you wouldn't mind adding the stuff you've done in #11 to a wiki page in your fork, and then commenting here when you've done it, I'll merge it in as the first contribution.

Also if anyone wants to take ownership of writing a page on deployment from start to finish, feel free @fuseumass/tech-team

**Comment by Tookmund** [comment]
Done
https://github.com/ACMWM/hackerforce/wiki/Environment-Variables

**Comment by jwoglom** [comment]
Merged -- https://github.com/fuseumass/hackerforce/wiki/Environment-Variables
